package.json

```diff
@@ -201,7 +201,7 @@
         "key": "Ctrl+Enter",
         "mac": "Cmd+Enter",
         "command": "dbtPowerUser.executeSQL",
-        "when": "resourceLangId == jinja-sql"
+        "when": "editorTextFocus && resourceLangId == 'jinja-sql'"
```

Comment:
Why not `editorFocus`?

 https://code.visualstudio.com/api/references/when-clause-contexts

Do you think this should be applied to other commands as well?

Reply:
My thought is that I should actively be in a jinja sql textbox for ctrl+enter to use this command.  (since a lot of things use it)

for the sql preview one.  I don't have anything installed that uses ctrl+', so it's not like it will overwrite another command.  

The other ones I didn't change because if I'm selecting something from the menu, then I'm explicitly asking for it.  So if I'm doing that I don't want to have to be focused in a certain spot.

Reply:
Thinking about this a tiny bit more.  Wouldn't it make sense for the SQL preview to be the same as markdown preview
`ctrl+shift+v` (on windows)?

Reply:
What's the behavior if you change to `editorFocus`, if it also fixes the issue, it would capture the user intent in more cases.

Reply:
Fair enough.  It still fixes the issue.